### ISSUE TITLE
Can't install because of composer requirements

### ISSUE THREAD

**Issue by AndrewMac** [spicywebau/commerce-afterpay#4]
Hi there SpicyWeb!

I guess the title says it all ... I can't install the plugin because the requirements can't be resolved.
If I understand things correctly, it seems that the package is requiring an old version of craftcms/commerce-omnipay
https://packagist.org/packages/craftcms/commerce-omnipay
That package is now at 3.0.3.1, and the AfterPay plugin requires ^2.0.0
Any chance you could look into that when you have the time? Thanks so much!

```
nitro composer require spicyweb/commerce-afterpay
  … checking /Users/path/to/composer.json ✓
Using version ^1.0 for spicyweb/commerce-afterpay
./composer.json has been updated
Running composer update spicyweb/commerce-afterpay
>
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires spicyweb/commerce-afterpay ^1.0 -> satisfiable by spicyweb/commerce-afterpay[1.0.0].
    - spicyweb/commerce-afterpay 1.0.0 requires craftcms/commerce-omnipay ^2.0.0 -> found craftcms/commerce-omnipay[2.0.0, ..., 2.1.1] but the package is fixed to 3.0.3.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by ttempleton** [comment]
Thanks for reporting this - turns out there was an issue that prevented the last release from being picked up by Packagist. Just released version 1.1.1 to fix this.

**Comment by AndrewMac** [comment]
Thanks so much!
